### ISSUE TITLE
Accessibility concerns

### ISSUE THREAD

**Issue by xtine** [eregs/notice-and-comment#166]
Current N&C is not very accessible, if we are following the Accessibility Guide checklist: https://pages.18f.gov/accessibility/checklist/

Main concern is around keyboard access. Because the site has limited functionality without JS (#164), there is no way to switch from read to write modes, or to write a comment since those actions are triggered by JS click events. If we make the site more accessible without JS (make anchor links to write comment pages as a fallback), it would also make keyboard navigation to those sections possible.
https://pages.18f.gov/accessibility/keyboard/

**Comment by cmc333333** [comment]
While I, too, don't want to require JS, let's not conflate accessibility with non-JS support. Using anchor tags in place of the existing elements (perhaps with tabindexes to change order) should make them keyboard accessible. We should have been using anchors from the start, but it's a much easier path to swapping them in than to add non-JS support.

**Comment by xtine** [comment]
I agree, the issues are related but not the same.

**Comment by xtine** [comment]
First steps:
* Tabindex and link roles for read/write tabs, and write a comment links
* Bind enter key press to click events so tabbing and pressing enter will trigger tabs and comment links.

**Comment by cmc333333** [comment]
Won't swapping out the `<span>`s and `<div>`s for `<a>` resolve the click problem? Might even consider `<button>`s. As a bonus anchors, buttons, etc. are much more semantically accurate.

**Comment by xtine** [comment]
As far as I tried (and researched), pressing enter with focus on an element isn't the same as a click event even if they are anchor tags. Haven't tried `<button>` yet.

**Comment by cmc333333** [comment]
Try an anchor with an href; the enter key is how links are clicked when using the keyboard.

**Comment by cmc333333** [comment]
Sorry about that!

**Comment by xtine** [comment]
Anchor with href still won't activate a click when there is focus on it, because just the click event that captures the changing of the content area.

**Comment by cmc333333** [comment]
Unless I'm completely mistaken, the "click" event is emitted when a user presses enter on a link. Try out the code in eregs/regulations-site#266 to see this in action.

**Comment by xtine** [comment]
Woahhh my bad I guess all the things I tried didn't work because typically I don't use empty hrefs. Although probably empty links aren't the best practice, we should probably go forward with this unless we have a better solution overall.

**Comment by cmc333333** [comment]
@xtine ran an audit and we're in a good place!
